lib/vagrant-vbguest/installers/debian.rb

```diff
@@ -13,12 +13,8 @@ def self.match?(vm)
       # @yieldparam [String] type Type of the output, `:stdout`, `:stderr`, etc.
       # @yieldparam [String] data Data for the given output.
       def install(opts=nil, &block)
-        begin
-          communicate.sudo(install_dependencies_cmd, opts, &block)
-        rescue
-          communicate.sudo('apt-get update', opts, &block)
-          communicate.sudo(install_dependencies_cmd, opts, &block)
-        end
+        communicate.sudo('apt-get update', opts, &block)
```

Comment:
Let's avoid of stopping because of apt-get failed to run (404 repos causing that)
add ```|| true``` or ```-qq``` as an argument